shiny/ui/_chat.py

```diff
@@ -1376,6 +1669,53 @@ def ui(
             **kwargs,
         )
 
+    def enable_bookmarking(
+        self,
+        client: ClientWithState | chatlas.Chat[Any, Any],
+        /,
+        *,
+        store: Optional[BookmarkStore] = None,
+        on: Optional[Literal["response"]] = "response",
+    ) -> CancelCallback:
+        """
+        Enable bookmarking for the chat instance.
+
+        This method registers `on_bookmark` and `on_restore` hooks on `session.bookmark`
+        to save/restore chat state on both the `Chat` and `client` instances.
+        In order for this method to actually work correctly, a `bookmark_store`
```

Comment:
Would be nice if these linked to the reference (i.e., `:class:`~shiny.express.session.bookmark``)

Reply:
Maybe this works :meth:`~shiny.session.ClientData.url_hash`?